Change default resize resampling filter from NEAREST to BICUBIC (#4255)
python-pillow/Pillow: docs/releasenotes/7.0.0.rst

Setting the size of a TIFF image directly (eg. ``im.size = (256, 256)``) throws
an error. Use ``Image.resize`` instead.

Default resampling filter
^^^^^^^^^^^^^^^^^^^^^^^^^

> The default resampling filter is changed to high-quality convolution

Reviewer: ```suggestion
The default resampling filter has been changed to the high-quality convolution
```
Just a suggestion.

> ``Image.BICUBIC`` instead of ``Image.NEAREST`` for the :py:meth:`~PIL.Image.Image.resize`

Reviewer: ```suggestion
``Image.BICUBIC`` instead of ``Image.NEAREST``, for the :py:meth:`~PIL.Image.Image.resize`
```

> method and for :py:meth:`~PIL.ImageOps.pad``, :py:meth:`~PIL.ImageOps.scale``

Reviewer: ```suggestion
method and the :py:meth:`~PIL.ImageOps.pad``, :py:meth:`~PIL.ImageOps.scale``
```
Just a suggestion.

and :py:meth:`~PIL.ImageOps.fit`` functions.
``Image.NEAREST`` is still always used for images in "P" and "1" modes.
See :ref:`concept-filters` to learn the difference. In short,

> ``Image.NEAREST`` is a very fast and simple but low-quality filter.

Reviewer: ```suggestion
``Image.NEAREST`` is a very fast filter, but simple and low-quality.
```
